silkrpc/core/remote_buffer.cpp

```diff
@@ -81,25 +81,49 @@ asio::awaitable<std::optional<evmc::bytes32>> AsyncRemoteBuffer::canonical_hash(
 
 std::optional<silkworm::Account> RemoteBuffer::read_account(const evmc::address& address) const noexcept {
     SILKRPC_DEBUG << "RemoteBuffer::read_account address=" << address << " start\n";
-    std::future<std::optional<silkworm::Account>> result{asio::co_spawn(io_context_, async_buffer_.read_account(address), asio::use_future)};
-    const auto optional_account{result.get()};
-    SILKRPC_DEBUG << "RemoteBuffer::read_account account.nonce=" << (optional_account ? optional_account->nonce : 0) << " end\n";
-    return optional_account;
+    try {
+        std::future<std::optional<silkworm::Account>> result{asio::co_spawn(io_context_, async_buffer_.read_account(address), asio::use_future)};
+        const auto optional_account{result.get()};
+        SILKRPC_DEBUG << "RemoteBuffer::read_account account.nonce=" << (optional_account ? optional_account->nonce : 0) << " end\n";
+        return optional_account;
+    } catch (const std::exception& e) {
+        SILKRPC_ERROR << "RemoteBuffer::read_account exception: " << e.what() << "\n";
+        return std::nullopt;
+    } catch (...) {
+        SILKRPC_ERROR << "RemoteBuffer::read_account unknown exception\n";
+        return std::nullopt;
+    }
```

Comment:
You could remove `catch (...)` and install a custom `std::termination_handler` to trap the program termination and emit a log trace before aborting